release-notes/docs/3.13.0.md

```diff
@@ -0,0 +1,85 @@
+# 3.13.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Breaking changes](#breaking-changes)
+- [UI/UX changes](#uiux-changes)
+- [Highlights](#highlights)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+- [#7235](https://github.com/medic/cht-core/issues/7235): Make it possible to display previous task due date formatting via configuration
+
+
+## Highlights
+
+### Collect form data from external Android apps
+
+The CHT now supports forms that collect data from generic 3rd-party Android apps. Users can launch the apps from the form and then the output from the app is captured by the CHT and stored with the report. See the [CHT documentation](https://docs.communityhealthtoolkit.org/apps/reference/forms/app/#android-app-launcher) for more information on how to use this feature.
+
+> One example use case for this functionality is to integrate a COVID-19 configuration with a rapid diagnostic testing app (such as [RD-Toolkit](https://github.com/dimagi/rd-toolkit/)) to allow capturing the results of the RTD in the CHT report.
```

Comment:
I literally (as in 30 seconds ago) just merged the PR in docs w/ [the COVID-19 config docs](https://docs.communityhealthtoolkit.org/apps/examples/covid-rdt-reference-app/).  We should likely cross reference this since it's a more explicit implementation of the 3rd party integration. Not blocking though!

Reply:
Ah perfect!  I was looking at that PR this morning, but saw that it was not merged yet.  I will update this to link to that doc.